PORKBUN: Fix staticcheck warnings

Hi @imlonghao!

It's not urgent, but would you please correct these when you have time.

Thanks!


```
$ staticcheck ./...
...
providers/porkbun/api.go:155:27: method createUrlForwardingRecord should be createURLForwardingRecord (ST1003)
providers/porkbun/api.go:162:27: method deleteUrlForwardingRecord should be deleteURLForwardingRecord (ST1003)
providers/porkbun/api.go:170:27: method modifyUrlForwardingRecord should be modifyURLForwardingRecord (ST1003)
providers/porkbun/api.go:180:27: method getUrlForwardingRecords should be getURLForwardingRecords (ST1003)
providers/porkbun/api.go:194:2: should replace loop with records = append(records, dr.Forwards...) (S1011)
```

--- a/providers/porkbun/api.go
+++ b/providers/porkbun/api.go
@@ -152,14 +152,14 @@ func (c *porkbunProvider) getRecords(domain string) ([]domainRecord, error) {
 	return records, nil
 }
 
-func (c *porkbunProvider) createUrlForwardingRecord(domain string, rec requestParams) error {
+func (c *porkbunProvider) createURLForwardingRecord(domain string, rec requestParams) error {
 	if _, err := c.post("/domain/addUrlForward/"+domain, rec); err != nil {
 		return fmt.Errorf("failed create url forwarding record (porkbun): %w", err)
 	}
 	return nil
 }
 
-func (c *porkbunProvider) deleteUrlForwardingRecord(domain string, recordID string) error {
+func (c *porkbunProvider) deleteURLForwardingRecord(domain string, recordID string) error {
 	params := requestParams{}
 	if _, err := c.post(fmt.Sprintf("/domain/deleteUrlForward/%s/%s", domain, recordID), params); err != nil {
 		return fmt.Errorf("failed delete url forwarding record (porkbun): %w", err)
@@ -167,17 +167,17 @@ func (c *porkbunProvider) deleteUrlForwardingRecord(domain string, recordID stri
 	return nil
 }
 
-func (c *porkbunProvider) modifyUrlForwardingRecord(domain string, recordID string, rec requestParams) error {
-	if err := c.deleteUrlForwardingRecord(domain, recordID); err != nil {
+func (c *porkbunProvider) modifyURLForwardingRecord(domain string, recordID string, rec requestParams) error {
+	if err := c.deleteURLForwardingRecord(domain, recordID); err != nil {
 		return err
 	}
-	if err := c.createUrlForwardingRecord(domain, rec); err != nil {
+	if err := c.createURLForwardingRecord(domain, rec); err != nil {
 		return err
 	}
 	return nil
 }
 
-func (c *porkbunProvider) getUrlForwardingRecords(domain string) ([]domainRecord, error) {
+func (c *porkbunProvider) getURLForwardingRecords(domain string) ([]domainRecord, error) {
 	params := requestParams{}
 	var bodyString, err = c.post("/domain/getUrlForwarding/"+domain, params)
 	if err != nil {
@@ -190,11 +190,7 @@ func (c *porkbunProvider) getUrlForwardingRecords(domain string) ([]domainRecord
 		return nil, fmt.Errorf("failed parsing url forwarding record list from porkbun: %w", err)
 	}
 
-	var records []domainRecord
-	for _, rec := range dr.Forwards {
-		records = append(records, rec)
-	}
-	return records, nil
+	return dr.Forwards, nil
 }
 
 func (c *porkbunProvider) getNameservers(domain string) ([]string, error) {

--- a/providers/porkbun/porkbunProvider.go
+++ b/providers/porkbun/porkbunProvider.go
@@ -146,7 +146,7 @@ func (c *porkbunProvider) GetZoneRecordsCorrections(dc *models.DomainConfig, exi
 				Msg: change.Msgs[0],
 				F: func() error {
 					if change.New[0].Type == "PORKBUN_URLFWD" {
-						return c.createUrlForwardingRecord(dc.Name, req)
+						return c.createURLForwardingRecord(dc.Name, req)
 					}
 					return c.createRecord(dc.Name, req)
 				},
@@ -161,7 +161,7 @@ func (c *porkbunProvider) GetZoneRecordsCorrections(dc *models.DomainConfig, exi
 				Msg: fmt.Sprintf("%s, porkbun ID: %s", change.Msgs[0], id),
 				F: func() error {
 					if change.New[0].Type == "PORKBUN_URLFWD" {
-						return c.modifyUrlForwardingRecord(dc.Name, id, req)
+						return c.modifyURLForwardingRecord(dc.Name, id, req)
 					}
 					return c.modifyRecord(dc.Name, id, req)
 				},
@@ -172,7 +172,7 @@ func (c *porkbunProvider) GetZoneRecordsCorrections(dc *models.DomainConfig, exi
 				Msg: fmt.Sprintf("%s, porkbun ID: %s", change.Msgs[0], id),
 				F: func() error {
 					if change.Old[0].Type == "PORKBUN_URLFWD" {
-						return c.deleteUrlForwardingRecord(dc.Name, id)
+						return c.deleteURLForwardingRecord(dc.Name, id)
 					}
 					return c.deleteRecord(dc.Name, id)
 				},
@@ -192,7 +192,7 @@ func (c *porkbunProvider) GetZoneRecords(domain string, meta map[string]string) 
 	if err != nil {
 		return nil, err
 	}
-	forwards, err := c.getUrlForwardingRecords(domain)
+	forwards, err := c.getURLForwardingRecords(domain)
 	if err != nil {
 		return nil, err
 	}

